integration/bundle/artifacts_test.go

```diff
@@ -1,4 +1,4 @@
-package bundle
+package bundle_integration
```

Comment:
Could you explain, why don't we call this package bundle_test?

Reply:
No particular reason, have to use some suffix.

Reply:
I didn't know that any tools used this convention for anything. I'll update.